presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionFactory.java

```diff
@@ -222,6 +228,12 @@ public Session createSession(
         return sessionProperties;
     }
 
+    private Set<String> parseTags(HttpServletRequest servletRequest)
+    {
+        Splitter splitter = Splitter.on(',').trimResults().omitEmptyStrings();
+        return ImmutableSet.copyOf(splitter.splitToList(nullToEmpty(servletRequest.getHeader(PRESTO_TAGS))));
```

Comment:
do you think we should have some cap on the number of tags that we allow to be set on http requests?

Reply:
It's sort of reasonable, but I don't know what's a reasonable number. 10? 50? 100? (10 seems to be to small though)